### PR TITLE
Move Tensor and related classes from System.Numerics to System.Numerics.Tensors

### PR DESCRIPTION
**Pull request by KonH**
Related to #1816

### CODE REVIEW

**Comment by KrzysztofCwalina** [comment]
LGTM.

File: src/System.Numerics.Tensors/System/Numerics/ArrayTensorExtensions.cs [hunk 1]
```diff
@@ -2,7 +2,7 @@
 // The .NET Foundation licenses this file to you under the MIT license.
 // See the LICENSE file in the project root for more information.
 
-namespace System.Numerics
+namespace System.Numerics.Tensors
```
Comment (eerhardt) [inline src/System.Numerics.Tensors/System/Numerics/ArrayTensorExtensions.cs]: Can you make a `Tensors` folder as well so the folders match the namespace?

**Comment by ericstj** [comment]
In addition to the namespace we also follow the folder convention for namsepace, so these files should be moved into a Tensors subfolder folder.

**Comment by KonH** [comment]
I added two more commits - now files moved and using directives in samples fixed.
@KrzysztofCwalina "LGTM" - do you mean that this PR is appropriate or may be I don't understand acroabbreviation and breaks some contribution rules? (unfortunately, I can't find detailed docs about it)

**Comment by eerhardt** [comment]
> LGTM

Stands for "Looks good to me".  😄

**Pull request merged**